app/src/controllers/bucket.js

```diff
@@ -74,9 +104,13 @@ const controller = {
   async createBucket(req, res, next) {
     const data = { ...req.body };
     let response = undefined;
+
+    // Check for credential accessibility/validity first
+    await this._validateCredentials(data, res);
```

Comment:
Do we want this call outside of try/catch?

Reply:
In this flow, it's leveraging the global catch resolver in app.js to handle the Problem object that has been thrown. The global catch resolver knows how to handle Problem object types and return a request appropriately, so a try/catch wrapper is not necessary here.

Reply:
Yeah exactly. The local catch could be used but then it would have to have logic to detect what the head bucket stuff throws (see notes in discussion about the different exception cases from that), or just rethrow the 409.
So think it's simpler and more maintainable to throw outside the one in the function, mostly because that local catch already does its own additional logic so don't like the catch block getting too big.

Doing the `await controller._validateCredentials(data, res);`